package.json

```diff
@@ -58,6 +58,7 @@
     "postman-sandbox": "4.2.3",
     "postman-url-encoder": "3.0.5",
     "serialised-error": "1.1.3",
+    "strip-json-comments": "3.1.1",
     "uuid": "8.3.2"
```

Comment:
I hope we are using 3.1.1 for common js support . Have we considered the other parameters like bugs, improvements, any security fixes , package size between 3.1.1  and 5.0.0

Reply:
Checked there were no bug fixes, just additional functionality of trimming comas and making this esm module.